externals/libarchive/src/configureHook.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+./configure \
+    --enable-static \
+    --disable-shared \
+    --disable-bsdtar \
+    --disable-bsdcat \
+    --disable-bsdcpio \
+    --without-zlib \
+    --without-bz2lib \
+    --without-lz4 \
+    --without-lzma \
+    --without-lzo2 \
+    --without-cng \
+    --without-nettle \
+    --without-openssl \
+    --without-xml2 \
+    --without-expat \
+    --disable-maintainer-mode \
```

Comment:
I forgot, what is this for?

Reply:
From the help: 
 --disable-maintainer-mode
                          disable make rules and dependencies not useful (and
                          sometimes confusing) to the casual installer

I was just removing as much as possible in order to make it as simple as possible to compile, however, digging deeper it does not seems like a good idea to keep it around.

It basically don't regenerate the Makefiles, configure scripts, etc when something changes in Makefile.in . It is pretty neutral for us, but I don't think is a good idea...

Removing it.